inference-engine/src/mkldnn_plugin/cpu_shape.h

```diff
@@ -13,15 +13,57 @@
 
 namespace MKLDNNPlugin {
 
+class Interval {
+public:
+    Interval() = default;
+
+    Interval(Dim val) {
+        minValue = val;
+        maxValue = val;
+    }
+
+    Interval(Dim minVal, Dim maxVal) {
+        minValue = minVal;
+        maxValue = maxVal;
+    }
+
+    Interval(const std::initializer_list<Dim>& shape) {
```

Comment:
what if initializer_list will contain > 2 elements? and do we really need this ctor? is not enough `Interval(Dim minVal, Dim maxVal) {` ?

Reply:
Removed